index.html

```diff
@@ -7,6 +7,7 @@
     <title>Vite + React + TS</title>
   </head>
   <body>
+    <script>window.global = window;</script>
```

Comment:
We can remove this script tag

Reply:
Unfortunately we cannot; Tamagui requires this. :(